content/_data/changelogs/weekly.yml

```
@@ -24937,13 +24937,14 @@
       - type: rfe
         category: developer
         pull: 6867
-        issue: 60866
+        issue: 71515
         authors:
           - daniel-beck
           - basil
         pr_title: "[JENKINS-60866][JENKINS-71515] Use `JSON#parse` to process `codemirror-config`
           argument"
         message: |-
+          Improve Content Security Policy compatibility by removing an <code>eval</code> call from JS.
           Developer: Require syntactically valid JSON snippet to be returned from <code>MarkupFormatter#getCodemirrorConfig</code> / provided to <code>codemirrorconfig</code> in <code>f:textarea</code>.
```

I think that the "Developer:" text is best at the start of the entry, rather than on the second sentence.
```suggestion
          Developer: Improve Content Security Policy compatibility by removing an <code>eval</code> call from JS.
          Require syntactically valid JSON snippet to be returned from <code>MarkupFormatter#getCodemirrorConfig</code> / provided to <code>codemirrorconfig</code> in <code>f:textarea</code>.
```